std/variant.d

```diff
@@ -472,6 +472,13 @@ private:
                 auto rhsPA = getPtr(&temp.store);
                 return compare(rhsPA, zis, selector);
             }
+            // am I comparable with 'null'
+            static if (__traits(compiles, () => A.init == null))
+            if (rhsType == typeid(null)) // is rhs 'null'
+            {
+                // cool! Then, am I 'null'?
```

Comment:
I understand that there is similar comments on `variant.d` but I think they either should be omitted for self explanatory statements or be declarative sentences instead.

Reply:
I followed the same syntax for coherence with the existent code and comments.

Reply:
In my opinion, we should get rid of such comments, at least the new ones. `git blame` tells me that similar comments were added 14 years ago and GitHub tell me that this was added without any review (not entirely sure if the development were here already). Contribution guidelines also suggest the following: "Use precise technical terms. Avoids colloquialisms.". Since the rest of the comments are unrelated to this PR, I can manage to fix the rest and create a separate one for further discussion, if you prefer.

Reply:
done